src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ConnectivityTests/ConnectionBehaviorTest.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.Data.SqlClient.ManualTesting.Tests
 {
     public class ConnectionBehaviorTest
     {
-        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
+        [CheckConnStrSetupFact]
```

Comment:
Why this change? CheckConnStrSetupFact was the old way of defining conditional facts. You had to create a new class for every different combination of checks. The left side is the newer style.

Reply:
I'm always confused which one is to be used, I'll rather change all the old styles at once to avoid future confusions.